2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_setup.py:_flush():71] setting env: {}
2022-09-10 15:27:14,436 WARNING MainThread:9702 [wandb_setup.py:_flush():71] could not find program at run.py
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_setup.py:_flush():71] setting login settings: {}
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_init.py:_log_setup():371] Logging user logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-10_15-27-13/wandb/run-20220910_152714-2koa8h7v/logs/debug.log
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-10_15-27-13/wandb/run-20220910_152714-2koa8h7v/logs/debug-internal.log
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_init.py:init():404] calling init triggers
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-09-10 15:27:14,436 INFO    MainThread:9702 [wandb_init.py:init():449] starting backend
2022-09-10 15:27:14,436 INFO    MainThread:9702 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-10 15:27:14,446 INFO    MainThread:9702 [backend.py:ensure_launched():199] starting backend process...
2022-09-10 15:27:14,452 INFO    MainThread:9702 [backend.py:ensure_launched():204] started backend process with pid: 9754
2022-09-10 15:27:14,453 INFO    MainThread:9702 [wandb_init.py:init():458] backend started and connected
2022-09-10 15:27:14,454 INFO    MainThread:9702 [wandb_init.py:init():519] updated telemetry
2022-09-10 15:27:14,467 INFO    MainThread:9702 [wandb_init.py:init():549] communicating current version
2022-09-10 15:27:15,873 INFO    MainThread:9702 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.13.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-10 15:27:15,873 INFO    MainThread:9702 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-09-10 15:27:16,249 INFO    MainThread:9702 [wandb_init.py:init():592] starting run threads in backend
2022-09-10 15:27:17,851 INFO    MainThread:9702 [wandb_run.py:_console_start():1816] atexit reg
2022-09-10 15:27:17,851 INFO    MainThread:9702 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-09-10 15:27:17,851 INFO    MainThread:9702 [wandb_run.py:_redirect():1695] Redirecting console.
2022-09-10 15:27:17,853 INFO    MainThread:9702 [wandb_run.py:_redirect():1751] Redirects installed.
2022-09-10 15:27:17,853 INFO    MainThread:9702 [wandb_init.py:init():619] run started, returning control to user process
2022-09-10 15:27:18,020 INFO    MainThread:9702 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-09-10 15:27:18,020 INFO    MainThread:9702 [wandb_run.py:_restore():1758] restore
2022-09-10 15:27:20,861 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-10 15:27:21,158 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-10 15:27:21,264 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7081
}

2022-09-10 15:27:21,365 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7081
}

2022-09-10 15:27:21,466 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7081
}

2022-09-10 15:27:21,567 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7081
}

2022-09-10 15:27:21,669 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2527
  total_bytes: 7081
}

2022-09-10 15:27:21,770 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6008
  total_bytes: 7081
}

2022-09-10 15:27:21,871 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:21,972 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:22,073 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:22,175 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:22,276 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:22,377 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:22,915 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}

2022-09-10 15:27:23,542 INFO    MainThread:9702 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7081
  total_bytes: 7081
}
local_info {
}

2022-09-10 15:27:24,656 INFO    MainThread:9702 [wandb_run.py:_append_files():2184] logging synced files
